File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
    long result = 0;
    int dummy = x;
    int digit;
    while(x > 0){
        digit = x % 10;
        result = result*10 + digit;
        x = x / 10;
    }
    if (result == dummy){
        return true;
    }
    else{
        return false;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.js
function isPalindrome(x) {
  let result = 0;
  let dummy = x;
  
  while (x > 0) {
    let digit = x % 10;
    result = result * 10 + digit;
    x = Math.floor(x / 10); // Use Math.floor for integer division
  }

  return result === dummy;
}
